src/main/java/dev/openfeature/sdk/EventSupport.java

```diff
@@ -1,5 +1,8 @@
 package dev.openfeature.sdk;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
I think this is the wrong import... not your change but can you fix it?

Reply:
yup, will fix that to have consistency across other usages

Reply:
done :)